web/settings/modulconfigpv.php

```diff
@@ -277,7 +277,25 @@
 								<div class="col">
 									<input class="form-control" type="number" name="solarview_port" id="solarview_port" value="<?php echo htmlspecialchars($solarview_portold) ?>">
 									<span class="form-text small">
-										Gültige Werte Port, z.B. 15000.
+										Gültige Werte: Port, z.B. 15000
+									</span>
+								</div>
+							</div>
+							<div class="form-row mb-1">
+								<label for="solarview_timeout" class="col-md-4 col-form-label">Timeout für den Solarview TCP-Server</label>
+								<div class="col">
+									<input class="form-control" type="number" name="solarview_timeout" id="solarview_timeout" value="<?php echo htmlspecialchars($solarview_timeoutold) ?>">
```

Comment:
Der Parameter `solarview_timeout` muss noch in der `updateConfig.sh` ergänzt werden.

Reply:
Mir ist die Rolle der `updateConfig.sh` nicht ganz klar. Ich vermute, sie sorgt dafür, dass die Konfiguration alle notwendigen Parameter enthält. Notwendig ist aber nur der Host. Port, Timeout und Kommando sind optional. Gehören sie trotzdem in die `updateConfig.sh`?

Reply:
Ja, auch die optionalen Standardwerte gehören in die openwb.conf. Die updateConfig.sh ist das einzige Skript, was neue Zeilen in der openwb.conf anlegt. Werden im UI Änderungen gemacht, werden lediglich bereits vorhandene Einstellungen aktualisiert, jedoch keine neuen Zeilen ergänzt.

Reply:
OK, habe ich ergänzt. Die Default-Werte werden auf diese Weise dann aber leider an 2 Orten gepflegt, einmal im Code und einmal im Update-Skript.